app/navigation/TrusteeOrganisationNavigator.scala

```diff
@@ -54,5 +75,46 @@ class TrusteeOrganisationNavigator extends Navigator {
         yesCall = UkAddressController.onPageLoad(index, draftId),
         noCall = InternationalAddressController.onPageLoad(index, draftId)
       )
+    case page @ CountryOfResidenceYesNoPage(index) => ua =>
+      yesNoNav(
+        ua = ua,
+        fromPage = page,
+        yesCall = mld5.CountryOfResidenceInTheUkYesNoController.onPageLoad(index, draftId),
+        noCall = CheckDetailsController.onPageLoad(index, draftId)
+      )
+    case page @ CountryOfResidenceInTheUkYesNoPage(index) => ua =>
+      yesNoNav(
+        ua = ua,
+        fromPage = page,
+        yesCall = AddressYesNoController.onPageLoad(index, draftId),
+        noCall = mld5.CountryOfResidenceController.onPageLoad(index, draftId)
+      )
+  }
+
+  private def mld5NavUTRYesNo(draftId: String, index: Int, is5mldEnabled: Boolean): Call = {
+    if (is5mldEnabled) {
+      mld5.CountryOfResidenceYesNoController.onPageLoad(index, draftId)
+    } else {
+      CheckDetailsController.onPageLoad(index, draftId)
+    }
+  }
+
+  private def navigateAwayFromIncomeQuestions(draftId: String, index: Int, is5mldEnabled: Boolean): Call = {
+    if (is5mldEnabled) {
+      mld5.CountryOfResidenceYesNoController.onPageLoad(index, draftId)
+    } else {
+      AddressYesNoController.onPageLoad(index, draftId)
+    }
+  }
+
+  private def addressOrCheckAnswersRoute(draftId: String, index: Int, userAnswers: ReadableUserAnswers): Call = {
+    userAnswers.get(UtrYesNoPage(index)) match {
+      case Some(true) =>
+        CheckDetailsController.onPageLoad(index, draftId)
+      case Some(false) =>
+        AddressYesNoController.onPageLoad(index, draftId)
+      case None =>
+        controllers.routes.SessionExpiredController.onPageLoad()
+    }
```

Comment:
This uses the same logic as the `yesNoNav` method in the `Navigator` trait. Would recommend utilising that to avoid repeated code. So it will look something like

`yesNoNav(
        ua = ua,
        fromPage = UtrYesNoPage(index),
        yesCall = CheckDetailsController.onPageLoad(index, draftId),
        noCall = AddressYesNoController.onPageLoad(index, draftId)
)`